Client/Frontend/Browser/PageZoom/PageZoomSettingsView.swift

```diff
@@ -0,0 +1,172 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+import Data
+import CoreData
+
+extension Domain: Identifiable {
+  public var id: String {
+    url ?? UUID().uuidString
+  }
+}
+
+private extension Domain {
+  var host: String {
+    URL(string: url ?? "")?.host ?? ""
+  }
+}
+
+struct PageZoomSettingsView: View {
+  @Environment(\.managedObjectContext) private var context
+  
+  @State private var domains = PageZoomSettingsView.filteredDomains(DataController.swiftUIContext)
+  
+  @State private var defaultZoomLevel = Preferences.General.defaultPageZoomLevel.value
+  
+  var body: some View {
+    List {
+      if !domains.isEmpty {
+        Section(
+          header: Text(Strings.PageZoom.specificWebsitesZoomLevelsSectionTitle)
+            .font(.footnote)
+            .foregroundColor(Color(.secondaryBraveLabel))
+            .multilineTextAlignment(.leading)
```

Comment:
Default alignment will be `.leading`, this shouldn't be needed (same for 5 line below in this file it's used)